stable/artifactory-ha/templates/artifactory-node-statefulset.yaml

```diff
@@ -75,6 +75,18 @@ spec:
 {{ tpl .Values.artifactory.customInitContainers . | indent 6}}
     {{- end }}
       containers:
+      {{- range .Values.artifactory.loggers }}
+      - name: {{ . | replace "_" "-" }}
+        image: busybox
```

Comment:
Please get the image from a parameter. Also set a value with a specific tag. We don't like `latest` as a default.